elytron/ELY-1686-legacy-properties-files-into-FileSystemRealm-utility.adoc

```diff
@@ -0,0 +1,109 @@
+= ELY-1686 Add utility to port legacy properties files into a FileSystemRealm
+:author:            Justin Michael Cook
+:email:             jucook@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+Currently, converting legacy security properties files into an Elytron FileSystemRealm
+must be done manually, there should be a migration tool that automates this conversion.
+This task is to add such a tool to WildFly Elytron Tool.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/ELY-1686[ELY-1686]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1093[EAP7-1093]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:msvehla@redhat.com[Martin Švehla]
+
+=== Affected Projects or Components
+
+* WildFly Elytron Tool
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+
+* It should be possible to have legacy properties files converted into an Elytron
+FileSystemRealm automatically. This should be done by adding a command named
+FileSystemRealmCommand to the Elytron-Tool. This command will take in the name of
+the file or files to be converted as the only parameter. The command will parse the given
+legacy properties file to configure the new security domain, appropriate factories,
+appropriate subsystems, and http-interface if necessary. There should be two possible
```

Comment:
It would be good to give some more details on what specifically the command will do (e.g., it will parse a legacy properties file and create a ```FileSystemSecurityRealm``` with users/passwords from the legacy properties file, and then it will output Elytron subsystem commands that can be used to configure a ```filesystem-realm```, ```security-domain```, etc.).

Reply:
This analysis should also mention how roles properties files will be handled.